backend-hormonia/app/routers/auth_session.py

```
@@ -30,7 +31,10 @@
 from app.services import ServiceProvider
 from app.services.audit_log import AuditLogService
 from app.dependencies.auth_dependencies import _firebase_service, _get_service_provider
+from app.dependencies.simple_service_provider import get_simple_service_provider
```

Pylint (3.3.9)

[error] No name 'simple_service_provider' in module 'app.dependencies'

(E0611)